### PR TITLE
add inside tote recognition launch and connect to main program

### PR DESCRIPTION
**Pull request by knorth55**
MERGE AFTER #1645 and #1646

### CODE REVIEW

**Comment by knorth55** [comment]
video https://drive.google.com/file/d/0B5DV6gwLHtyJM2NfU3FHd1RoNlk/view?usp=sharing

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 1]
```diff
@@ -347,8 +347,19 @@
              (ros::spin-once))
       (ros::unsubscribe box-topic)
       (ros::unsubscribe com-topic)
-      (setq _objects-in-order-bin-boxes (send box-msg :boxes))
-      (setq _objects-in-order-bin-coms com-msg)))
+      (cond
+        ((and box-msg com-msg)
+         (ros::ros-info "[:recognize-objects-in-order-bin] arm: ~a get cpi msg" arm)
+         (setq obj-box (send box-msg :boxes))
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: `obj-box` becomes global variables.
Is this ok?

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 2]
```diff
@@ -347,8 +347,19 @@
...
+         (setq obj-coords
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: `obj-box` and `obj-coords` become global variables.
Is this ok?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: does not matter, but i fixed

**Comment by pazeshun** [comment]
I saw the video.
I began to think we need mid-point to approach object from straight above.

**Comment by pazeshun** [comment]
+1 about lisp code

**Pull request merged**